mlflow/entities/assessment.py

```diff
@@ -173,6 +173,16 @@ def to_dictionary(self):
             "span_id": self.span_id,
```

Comment:
We have an updated V3 data model for assessment with some new changes. Could we update the assessment format here to match or create an `assessment_v3` to facilitate it if necessary? E.g. `name` is now `assessment_name`, the timestamps use the `google.protobuf.timestamp` type instead of an int, and `error` was moved into `feedback` (among other items).

Reply:
https://github.com/mlflow/mlflow/pull/15311 addresses this :)

Reply:
@harupy I see that https://github.com/mlflow/mlflow/pull/15311 is moving the `error` field, but it doesn't handle the other differences yet. Are you planning to broaden the scope of that PR?

Reply:
@ShaylanDias I'll file another PR to match the field names to v3 assessment.